apps/android_deploy/app/src/main/java/org/apache/tvm/android/demo/MainActivity.java

```diff
@@ -224,108 +224,115 @@ protected void onPostExecute(Integer status) {
-                                int src_index = k + IMG_CHANNEL*m + IMG_CHANNEL*MODEL_INPUT_SIZE*l;
-                                imgRgbTranValues[dst_index] = imgRgbValues[src_index];
-                            }
-                        }
-                    }
-
-                    // get the function from the module(set input data)
-                    Log.i(TAG, "set input data");
-                    NDArray inputNdArray = NDArray.empty(new long[]{1, IMG_CHANNEL, MODEL_INPUT_SIZE, MODEL_INPUT_SIZE}, new TVMType("float32"));;
-                    inputNdArray.copyFrom(imgRgbTranValues);
-                    Function setInputFunc = graphRuntimeModule.getFunction("set_input");
-                    setInputFunc.pushArg(INPUT_NAME).pushArg(inputNdArray).invoke();
-                    // release tvm local variables
-                    inputNdArray.release();
-                    setInputFunc.release();
-
-                    // get the function from the module(run it)
-                    Log.i(TAG, "run function on target");
-                    Function runFunc = graphRuntimeModule.getFunction("run");
-                    runFunc.invoke();
-                    // release tvm local variables
-                    runFunc.release();
-
-                    // get the function from the module(get output data)
-                    Log.i(TAG, "get output data");
-                    NDArray outputNdArray = NDArray.empty(new long[]{1, 1000}, new TVMType("float32"));
-                    Function getOutputFunc = graphRuntimeModule.getFunction("get_output");
-                    getOutputFunc.pushArg(OUTPUT_INDEX).pushArg(outputNdArray).invoke();
-                    float[] output = outputNdArray.asFloatArray();
-                    // release tvm local variables
-                    outputNdArray.release();
-                    getOutputFunc.release();
-
-                    // display the result from extracted output data
-                    if (null != output) {
-                        int maxPosition = -1;
-                        float maxValue = 0;
-                        for (int j = 0; j < output.length; ++j) {
-                            if (output[j] > maxValue) {
-                                maxValue = output[j];
-                                maxPosition = j;
-                            }
-                        }
-                        processingTimeMs = SystemClock.uptimeMillis() - processingTimeMs;
-                        String label = "Prediction Result : ";
-                        label += labels.size() > maxPosition ? labels.get(maxPosition) : "unknown";
-                        label += "\nPrediction Time : " + processingTimeMs + "ms";
-                        mResultView.setText(label);
-                    }
-                    Log.i(TAG, "prediction finished");
+          if (null != graphExecutorModule) {
```

Comment:
Why ss the indent size here is 2 while everywhere else its 4?

Reply:
good catch @cnv1989, I had run git-clang-format directly rather than with the script, and it looks like it affected java files. reverted.